### PR TITLE
59. Update the node runners page colours and skin tones

### LINKED ISSUE #59: Update the node-runners page colours and skin tones.
## 📄 Context
Here are the `node runners` main page changes on colour, skin tones and fonts in Dark mode to comply with the new Cartesi brand. The application's structure did not change, only the look and feel.

**PS: Refer to the parent of this issue for references.**
PS²: Don't worry about the **header** and **footer**, because that is addressed in a separate issue #62 

## 📈 Subtasks

**PS: A highlight for the two `cases` below. observe the spacing and difference in colours in between blocks, meaning they are not glued.**

### Overall
- [x] Make sure the fonts are applied correctly based on the Figma description. i.e.  Where to use each font and what size and weight to be applied.
- [x] Make sure the components are round, like the design. **We're transitioning from a squared-based design to a round one.**
### Page sessions
- [x] For the page header area:
  - [x] apply new colour and font rules for page **title** and **subtitle**. 
  - [x] **when wallet is disconnected**: Apply the new colour and font rules in the alert and connect area.
  
### Case: when the user doesn't have either Staking Pool or Private node
- [x] Apply colours and font rules to the `Create a node or pool in steps` title.
- [x] Apply colours and font rules for the `Create a public pool` card.
  - [x] Update the icon colour
  - [x] Update the button colour 
  - [x] Apply the new colours for the tooltip content (e.g. background colour, font). 
- [x] Apply colours and font rules for the `Run a private node` card.
  - [x] Update the icon colour
  - [x] Update the button colour 
  - [x] Apply the new colours for the tooltip content (e.g. background colour, font). 
- [ ] **Add here any extra tasks you see fit [Optional]**

### Case: when there is a Staking Pool and/or Private Node created

- [x] For the `Pool` block
   - [x] Apply the new font rules to the title.
   - [x] Apply colours changes to the `create pool` button
   - [x] Update the look and feel of the **Table** (check on Figma for table colours reference)
- [x] For the `Private node` block
   - [x] Apply the new font rules to the title.
   - [x] Update the look and feel of the **Table** (check on Figma for table colours reference)

## 🎯 Definition of Done

-   [ ] Node runners' main page matches the design specs on Figma.

### PR DESCRIPTION
**Pull request by dandheedge**
Here is the screenshot after implement the new style on Node Runners pages

Fixes #59 

https://github.com/cartesi/explorer/assets/13269955/cfa19182-d459-434e-99e8-63c78c6387bd

### CODE REVIEW

File: apps/staking/src/styles/components/table.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const Table = {
     variants: {
         simple: {
             th: {
-                backgroundColor: 'gray.900',
+                backgroundColor: 'dark.gray.secondary',
```
Comment (brunomenezes) [inline apps/staking/src/styles/components/table.ts:16]: That `src/styles/*` inside the staking should not be used anymore. We should only reference the one coming from `@explorer/ui`. Do you mind cleaning that up and replacing it where it is used inside staking? That will probably fix a bug where the screen is not automatically starting on dark-mode, but light-mode instead.


cc: @nevendyulgerov (just to be aware)

**Review by brunomenezes — CHANGES_REQUESTED** [review]
@dandheedge, I notice a couple of leftovers. When I close the alerts, the one above the pool management area leaves that empty element after clicking either the `X` or the `don't show again` link. The other thing is the table's last column. The colour is a bit off compared to the situations demonstrated in Figma. It only becomes that lighter dark when the last column is "floating" for visibility, but if not they are all the same column.

### Implemented
image

### Figma

image

File: apps/staking/src/styles/components/table.ts (hunk 1 above)
Comment (dandheedge) [reply apps/staking/src/styles/components/table.ts:16]: @brunomenezes sure thing, I will cleaning and replacing it up

Comment (brunomenezes) [reply apps/staking/src/styles/components/table.ts:16]: > @brunomenezes sure thing, I will cleaning and replacing it up

For the clean-up, will you do it on this PR? I saw you did fix some of the things I mentioned above.

File: apps/staking/src/pages/_document.tsx [hunk 2]
```diff
@@ -10,8 +10,8 @@
 // PARTICULAR PURPOSE. See the GNU General Public License for more details.
 
 import { ColorModeScript } from '@chakra-ui/react';
-import NextDocument, { Html, Head, Main, NextScript } from 'next/document';
-import theme from '../styles/theme';
+import { theme } from '@explorer/ui';
```
Comment (dandheedge) [inline apps/staking/src/pages/_document.tsx:13]: @brunomenezes I already change the reference to `@explorer/ui` package

**Comment by brunomenezes** [comment]
Hey @dandheedge, need to do some rebase.

**Review by brunomenezes — APPROVED** [review]

**Pull request merged**